cli/lib/kontena/client.rb

```diff
@@ -321,14 +327,15 @@ def request(http_method: :get, path:'/', body: nil, query: {}, headers: {}, resp
       }.merge(host_options)
 
       request_options.merge!(response_block: response_block) if response_block
+      @last_request = request_options
```

Comment:
I would prefer to get rid of the `@last_response`, and avoid adding a new `@last_request`...